README.md

```diff
@@ -27,6 +29,15 @@ The porting guide is available on [this page](http://www.FreeRTOS.org/FreeRTOS-P
 
 ## Repository structure
 This repository contains the FreeRTOS-Plus-TCP repository and a number of supplementary libraries for testing/PR Checks.
+Below is the breakdown of what each directory contains:
+- tools
+    - This directory contains the tools and related files (CMock/uncrustify) required for tests/checks for the TCP source code.
+- tests
+    - This directory contains all the tests (unit tests and CBMC) and the dependencies ([FreeRTOS-Kernel](https://github.com/FreeRTOS/FreeRTOS-Kernel)/[Litani-port](https://github.com/awslabs/aws-build-accumulator)) the tests require.
+- portable
+    - This directory contains the portable files required to compile the FreeRTOS-Plus-TCP source code for different hardwares/compilers.
+- include
+    - The include directory has all 'core' the header files of FreeRTOS-Plus-TCP source.
```

Comment:
has all the 'core' header files of the FreeRTOS-PLUS-TCP source

Reply:
Oops! What was I thinking... Have updated the language.